src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +565,222 @@ subroutine DustEmission (bounds, &
-               wnd_frc_slt_dlt = 0.003_r8 * wnd_rfr_dlt * wnd_rfr_dlt
-               wnd_frc_slt = fv(p) + wnd_frc_slt_dlt
-            end if
+            wnd_rfr_thr_slt = u10(p) * wnd_frc_thr_slt / fv(p)     ! keep and use if we want the default Z03 scheme
 
+            
             ! the following comes from subr. flx_mss_hrz_slt_ttl_Whi79_get
             ! purpose: compute vertically integrated streamwise mass flux of particles
 
-            if (wnd_frc_slt > wnd_frc_thr_slt) then
-               wnd_frc_rat = wnd_frc_thr_slt / wnd_frc_slt
-               flx_mss_hrz_slt_ttl = cst_slt * forc_rho(c) * (wnd_frc_slt**3.0_r8) * &
-                    (1.0_r8 - wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) / grav
+            !if (wnd_frc_slt > wnd_frc_thr_slt) then! if using Zender's scheme, use fluid threshold for dust emission equation below
+            if (wnd_frc_slt > wnd_frc_thr_slt_it) then! if using Leung's scheme, use impact threshold for dust emission equation
+
+               !################### for Zender et al. (2003) scheme -dmleung ###########################
+               !################ uncomment the below block if want to use Z03 scheme ###################
+               !wnd_frc_rat = wnd_frc_thr_slt / wnd_frc_slt
+               !flx_mss_hrz_slt_ttl = cst_slt * forc_rho(c) * (wnd_frc_slt**3.0_r8) * &
+               !     (1.0_r8 - wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) * (1.0_r8 + wnd_frc_rat) / grav  ! Zender dust emission equation for emission flux
 
                ! the following loop originates from subr. dst_mbl
                ! purpose: apply land sfc and veg limitations and global tuning factor
                ! slevis: multiply flx_mss_hrz_slt_ttl by liqfrac to incude the effect 
                ! of frozen soil
 
-               flx_mss_hrz_slt_ttl = flx_mss_hrz_slt_ttl * lnd_frc_mbl(p) * mbl_bsn_fct(c) * &
-                    flx_mss_fdg_fct * liqfrac
+               !flx_mss_hrz_slt_ttl = flx_mss_hrz_slt_ttl * lnd_frc_mbl(p) * mbl_bsn_fct(c) * &
+               !     flx_mss_fdg_fct * liqfrac
+
+               !
+               !dst_slt_flx_rat_ttl = 100.0_r8 * exp( log(10.0_r8) * (13.4_r8 * mss_frc_cly_vld(c) - 6.0_r8) )
+               !flx_mss_vrt_dst_ttl(p) = flx_mss_hrz_slt_ttl * dst_slt_flx_rat_ttl
+               !########################################################################################
+
+               !################### for Leung et al. (2022) ################################################
+               !################ uncomment the below block if want to use Leung's scheme ###################
+
+               flx_mss_vrt_dst_ttl(p) = dst_emiss_coeff(p) * mss_frc_cly_vld(c) * forc_rho(c) * ((wnd_frc_slt**2.0_r8 - wnd_frc_thr_slt_it**2.0_r8) / wnd_frc_thr_slt_std) * (wnd_frc_slt / wnd_frc_thr_slt_it)**frag_expt  ! Leung et al. (2022) uses Kok et al. (2014) dust emission euqation for emission flux
```

Comment:
I found a typo here: `wnd_frc_thr_slt_std` should be `wnd_frc_thr_slt_it` in line 722.

Reply:
This change shouldn't change the answer much, as tested previously. Maybe I should test it again though.

Reply:
This was already handled by the latest code.